docs/interfaces.md

```diff
@@ -249,6 +249,13 @@ Can configure network firewalling giving privileged access to networking.
 
 * Auto-Connect: no
 
+### fuse
+
+Can mount fuse filesystems (as root only).
```

Comment:
This says 'as root only', but down below you say 'Can run an unprivileged FUSE filesystem.'. These descriptions seem at odds.